client/client.go

```diff
@@ -261,8 +261,7 @@ func (c *Client) downloadTimestamp() error {
 	}
 	// unlike root, targets and snapshot, always try and download timestamps
 	// from remote, only using the cache one if we couldn't reach remote.
-	raw, err := c.remote.GetMeta(role, maxSize)
-	var s *data.Signed
+	raw, s, err := c.downloadSigned(role, maxSize, nil)
 	if err != nil || len(raw) == 0 {
```

Comment:
Will `len(raw)` ever be 0 here?  Won't that cause json unmarshalling to return an error (unexpected end of JSON input)?  If so, I don't think https://github.com/endophage/gotuf/pull/49/files#diff-1cd41a2611da49ce63630d5a51bd82c5L270 will ever be reached.